src/main/java/core/basesyntax/service/ExpectedException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+public class ExpectedException extends RuntimeException {

```

Comment:
Try to give more suitable names
```suggestion
public class RegistrationException extends RuntimeException {

```